sot/starcheck
# Get the ACA thermal model with get_xija_model_spec

## Description

Get the ACA thermal model with get_xija_model_spec

There's a little duplication here but since the proseco code has the convenient limits functionality, I left it alone.

## Testing

- [x] Functional testing - Probably needs more testing but doesn't error-out and makes a ccd temperature plot, so looks like it is getting the right model.

### Functional testing

I ran AUG3021A.  For the flight outputs, I reran flight starcheck so I wouldn't need to play with the run_start_time stuff.

The new top matter looks like
```
 ------------  Starcheck 13.11.1.dev10+gd4bae3e    -----------------
 Run on Tue Aug 31 11:43:46 EDT 2021 by jeanconn from fido.cfa.harvard.edu
 Configuration:  Using AGASC at /proj/sot/ska3/flight/data/agasc/proseco_agasc_1p7.h5
 chandra_models version: 3.37
```
in the HTML and txt.  
*Flight model plot*
<img src="https://user-images.githubusercontent.com/791508/131535374-b102570d-4aa0-4666-8894-4a709e88a05a.png" width=600>
*Test model plot*
<img src="https://user-images.githubusercontent.com/791508/131535404-c0e323d3-88c7-45d1-9bac-90db110bf8b9.png" width=600>

Diffs flight vs test are expected
```
jeanconn-fido> diff -u aug3021a.txt /home/jeanconn/git/starcheck/aug3021a.txt
--- aug3021a.txt	2021-08-31 11:53:40.739284000 -0400
+++ /home/jeanconn/git/starcheck/aug3021a.txt	2021-08-31 11:43:48.161616000 -0400
@@ -1,12 +1,13 @@
- ------------  Starcheck 13.11.0    -----------------
- Run on Tue Aug 31 11:53:38 EDT 2021 by jeanconn from fido.cfa.harvard.edu
+ ------------  Starcheck 13.11.1.dev10+gd4bae3e    -----------------
+ Run on Tue Aug 31 11:43:46 EDT 2021 by jeanconn from fido.cfa.harvard.edu
  Configuration:  Using AGASC at /proj/sot/ska3/flight/data/agasc/proseco_agasc_1p7.h5
+ chandra_models version: 3.37
 
 Short Term Schedule: AUG3021A
 
 ------------  PROCESSING FILES  -----------------
 
-DATA = /proj/sot/ska3/flight/lib/python3.8/site-packages/starcheck/data
+DATA = /proj/sot/ska/jeanproj/git/starcheck/starcheck/data
 Using DOT file /data/mpcrit1/mplogs/2021/AUG3021/oflsa/mps/mdAUG3021A.dot
 Using OR file /data/mpcrit1/mplogs/2021/AUG3021/oflsa/mps/or/AUG3021_A.or
 Using TLR file /data/mpcrit1/mplogs/2021/AUG3021/oflsa/CR242_0101.tlr
```
